app/javascript/packs/naics-bar-chart-race.js

```diff
@@ -0,0 +1,191 @@
+
+window.addEventListener('DOMContentLoaded', () => {
+  Promise.all([
+    d3.csv('/data/three-digit-sub-sector-01-17.csv', (d) => ({
+      measurementDate: +d.date,
+      categorys: d.category,
+      municipality: d.name,
+      employees: d.employees
```

Comment:
This should be updated to `+d.employees` to coerce it into the correct integer type to fix the issue with the type conversion that was causing the incorrect rankings.